Sources/Segment/Plugins/SegmentDestination.swift

```diff
@@ -156,6 +140,29 @@ public class SegmentDestination: DestinationPlugin {
     }
 }
 
+// MARK: - Utility methods
+extension SegmentDestination {
+    internal func configureCloudDestinations<T: RawEvent>(event: T) -> T {
+        guard let integrationSettings = analytics?.settings() else { return event }
+        guard let plugins = analytics?.timeline.plugins[.destination]?.plugins as? [DestinationPlugin] else { return event }
+        guard let customerValues = event.integrations?.dictionaryValue else { return event }
+        
+        // take the customer values first.
+        var merged = customerValues
```

Comment:
i think we should use integrationSettings as the base value, and then add the customer values to override here